ert_shared/storage/main.py

```diff
@@ -55,7 +55,7 @@ def bind_socket(host: str, port: int) -> socket.socket:
 
 
 def bind_open_socket(host: str) -> socket.socket:
-    for port in range(51820, 51840):
+    for port in range(51820, 51840 + 1):
```

Comment:
Do you mean `range(51820, 51841)` or `range(51820, 51840, 1)` or do you actually wants this `51740 + 1` ? And if that is the case, do you care to explain?

Reply:
Our ports are `51820 -- 51840` inclusive, however the Python `range` is a half-open interval:
```
>>> 51840 in range(51820, 51840)
False
>>> 51840 in range(51820, 51840 + 1)
True
```

Reply:
I write `51840 + 1` rather than `51841` because I think it conveys better that we want a closed interval.